### PR TITLE
LPS-180856 Fix TreeView styles

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
This is a follow-up of https://github.com/liferay-echo/liferay-portal/pull/11636
It fixes the styles of the tree so that actions do not overlap with the node name and the horizontal scroll is applied correctly.
[Link to task](https://issues.liferay.com/browse/LPS-151678)
[Link to subtask](https://issues.liferay.com/browse/LPS-180856)

Proposed Solution
========
We have changed the CSS

Steps to Verify:
----------------

1. Set feature.flag.LPS-151678=true
2. Go to home page and click on edit
3. In the left sidebar click on Browser section
4. Open the tree nodes and check that the styles are good
5. Set feature.flag.LPS-151678=false, restart the server and check the everything works as before

Screenshots ():
-----------------------

Captura de pantalla 2023-04-12 a las 11 27 52

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by claraizquierdo** [comment]
ci:test:sf

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:sf

**Review by victorg1991 — CHANGES_REQUESTED** [review]
Hey @claraizquierdo This looks fantastic 👏 

Just leave a couple of question/suggestions for you to take a look :) Let me know what do you think

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/_StructureTreeNode.scss [hunk 1]
```diff
@@ -4,6 +4,59 @@ $treeNodeNameMappedColor: #954cff;
...
+			.page-editor__page-structure__tree-node__actions-button,
+			.page-editor__page-structure__tree-node__visibility-button {
+				color: $cadmin-secondary;
+				margin-left: 0;
+				transform: translateY(1px);
+				visibility: hidden;
+
+				&--visible {
+					visibility: visible;
+				}
```
[the remark this answers is not in the document]
Comment (claraizquierdo) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/_StructureTreeNode.scss:56]: Yeah, all those styles are repeated and we will delete the old ones when we remove the FF. I'm going to have a look at the transform 🤔

**Comment by victorg1991** [comment]
Niiiiiiiiiiiice, let's merge this 👏

**Comment by victorg1991** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward:force

**Pull request closed without merging**